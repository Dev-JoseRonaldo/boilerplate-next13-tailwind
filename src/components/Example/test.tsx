import { render, screen } from '@testing-library/react'

import { Example } from '.'

const { getByRole } = screen

describe('<Example />', () => {
  it('should render the heading', () => {
    render(<Example title="Hello World" />)

    const heading = getByRole('heading', { name: 'Hello World' })

    expect(heading).toBeInTheDocument()
  })

  it('Should match snapshot', () => {
    const { container } = render(<Example title="Hello World" />)

    expect(container).toMatchSnapshot()
  })
})
