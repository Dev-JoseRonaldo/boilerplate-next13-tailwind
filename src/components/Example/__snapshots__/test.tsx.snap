// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Example /> Should match snapshot 1`] = `
<div>
  <div
    class="flex justify-center"
  >
    <h1
      class="font-bold text-lg"
    >
      Hello World
    </h1>
  </div>
</div>
`;
